### ISSUE TITLE
Accuracy issue on DFEW dataset

### ISSUE THREAD

**Issue by TopMhill** [sunlicai/MAE-DFER#14]
[nohup_pretrain_eval.docx](https://github.com/sunlicai/MAE-DFER/files/14428186/nohup_pretrain_eval.docx)
Hello, dear author!
I must express my admiration for your incredibly successful work. Inspired by your achievements, I have attempted to reproduce the results of your paper. I used the fine-tuned model parameters for DFEW that you uploaded on GitHub and conducted tests on folder 1 of the DFEW dataset. However, the results I obtained were UAR: 46.03% and WAR: 50.19%. I have attached the relevant log files in the appendix for your reference.
I also tried downloading your pre-trained model and performed fine-tuning on folder 1 of the DFEW dataset. However, the final accuracy results were only UAR: 54.75% and WAR: 67.24%. I would greatly appreciate your guidance on identifying potential mistakes or areas where we may have gone wrong. Any suggestions you could provide would be invaluable.Thank you very much for your time and assistance!

**Comment by youcaiSUN** [comment]
Could you provide more details?  Such as your running scripts and your processed label files on each fold. BTW, do you use temporal aligned frames (16) or the original extracted frames?

**Comment by 1162431418** [comment]
I have attempted to reproduce the results of your paper. I used the fine-tuned model parameters for DFEW that you uploaded on GitHub and conducted tests on folder 1 of the DFEW dataset. However, the results I obtained were UAR: 41.89% and WAR: [50.11%.]
I would greatly appreciate your guidance on identifying potential mistakes or areas where we may have gone wrong. Any suggestions you could provide would be invaluable.Thank you very much for your time and assistance!

nohup.txt is the relevant log file.
dfew_getoriginalframes.txt is the python code to get original frames(resize 256).

[nohup.txt](https://github.com/user-attachments/files/17656324/nohup.txt)
[dfew_getoriginalframes.txt](https://github.com/user-attachments/files/17656352/dfew_getoriginalframes.txt)

**Comment by sunlicai** [comment]
Hi @1162431418, thanks for your interest in our work. 

It seems that you extract the whole frame from the orignial video clip in [dfew_getoriginalframes.txt](https://github.com/user-attachments/files/17656352/dfew_getoriginalframes.txt). This is totally different from the common practice in DFER. You should use the **extracted faces** provided by the DFEW dataset instead of the whole frame as the latter includes much background information. The extracted faces can be found in the folder 'clip_224x224'. Note that I renamed it to 'jpg_256' because the real image resolution is 256x256. Then you can follow [dfew.py](https://github.com/sunlicai/MAE-DFER/blob/master/preprocess/dfew.py) to preprare the data.

**Comment by 1162431418** [comment]
> Hi @1162431418, thanks for your interest in our work.
> 
> It seems that you extract the whole frame from the orignial video clip in [dfew_getoriginalframes.txt](https://github.com/user-attachments/files/17656352/dfew_getoriginalframes.txt). This is totally different from the common practice in DFER. You should use the **extracted faces** provided by the DFEW dataset instead of the whole frame as the latter includes much background information. The extracted faces can be found in the folder 'clip_224x224'. Note that I renamed it to 'jpg_256' because the real image resolution is 256x256. Then you can follow [dfew.py](https://github.com/sunlicai/MAE-DFER/blob/master/preprocess/dfew.py) to preprare the data.

Thanks！I succeeded in replicating the code，tests on folder 1 ，UAR: 62.22%, WAR: 75.18%.

**Comment by sunlicai** [comment]
Happy to hear that!
